app/assets/scripts/components/card-list.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { PropTypes as T } from 'prop-types';
+import styled, { css } from 'styled-components';
+
+const Wrapper = styled.div`
+  display: grid;
+  ${({gridGap, gridTemplateRows, gridTemplateColumns}) => css`
+    grid-template-columns: ${gridTemplateColumns || 'repeat(4, 1fr)'};
+    grid-template-rows: ${gridTemplateRows || 'auto'};
+    grid-gap: ${gridGap || '2rem'};
```

Comment:
:+1: I like this approach of default styles that can easily be overridden if needed.